README.md

```diff
@@ -4,115 +4,31 @@
 
 <h1><p align="center">surface-dynamics</p></h1>
 
+<p align="center">
+  <img src="https://img.shields.io/badge/License-GPL_2.0_or_later-blue.svg" alt="License: GPL 2.0 or later">
+  <a href="https://github.com/flatsurf/surface-dynamics/actions/workflows/test.yml"><img src="https://github.com/flatsurf/surface-dynamics/actions/workflows/test.yml/badge.svg" alt="Test"></a>
+  <a href="https://doi.org/10.5281/zenodo.13356803"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.13356803.svg" alt="DOI 10.5281/zenodo.13356803"></a>
+</p>
+
+<p align="center">TODO</p>
```

Comment:
@videlec can you propose a short tagline that describes this project here?

Reply:
"Dynamics with surfaces"?